lib/utils/config/definitions.js

```diff
@@ -214,12 +214,14 @@ define('also', {
 
 define('audit', {
   default: true,
-  type: Boolean,
+  hint: '<pkg>',
+  type: [Boolean, String],
   description: `
     When "true" submit audit reports alongside the current npm command to the
-    default registry and all registries configured for scopes.  See the
-    documentation for [\`npm audit\`](/commands/npm-audit) for details on what
-    is submitted.
+    default registry and all registries configured for scopes. When passed a
+    dependency name an audit will be submitted for only that dependency. See
+    the documentation for [\`npm audit\`](/commands/npm-audit) for details on
+    what is submitted.
```

Comment:
the changes to the `audit` config here should be reverted, this is for the `--audit` flag that commands like `npm install` accept.

rather, i think we should piggyback on the `package` flag where all we should have to do is modify the description to mention how it's used for `npm audit`

Reply:
I removed this change from the `audit` config, but I wasn't able to find where the `package` flag's description is in order to update it there.

Reply:
https://github.com/npm/cli/blob/release-next/lib/utils/config/definitions.js#L1404